android/app/src/main/java/org/fossasia/openevent/utils/NetworkStateReceiver.java

```diff
@@ -0,0 +1,70 @@
+package org.fossasia.openevent.utils;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by arpitdec5 on 13-03-2017.
+ */
+
+public class NetworkStateReceiver extends BroadcastReceiver {
```

Comment:
@harshitagupta30
i think there is no need. And its better the mentors have a look at it and review.